!GameTexts/NewTexts.xml

```diff
@@ -91,32 +84,35 @@
+			<Text>Unique Siamese replacement for the Workshop.[NEWLINE][NEWLINE]
+				+1 [ICON_GOLD] Gold and +1 [ICON_PRODUCTION] Production in City for every 2 Strategic Resource received as City-State gifts.  Receive 10 [ICON_PRODUCTION] Production and [ICON_CULTURE] Culture in Capital for every Baan Chang in the Empire every time a city state gifts a unit, scales with Era.[NEWLINE][NEWLINE]
+				Allows [ICON_PRODUCTION] Production to be moved from this city along Trade Routes inside your civilization.</Text>
 		</Row>
 		<Row Tag="TXT_KEY_BUILDING_SIAM_BAAN_CHANG_STRATEGY">
 			<Text>The Baan Chang is the Siamese unique building, replacing the Workshop. In addition to the normal bonuses of a Workshop, The Baan Chang increases Gold from lumber mills, Production and Faith from Ivory. City state gifts, both Strategic Resources and units, provide additional benefits in cities with a Baan Chang, so be sure to make strong alliances to get the most from this building.</Text>
 		</Row>
 		<Row Tag="TXT_KEY_BUILDING_SIAM_BAAN_CHANG_TEXT">
-			<Text>Everybody loves elephants.</Text>
+			<Text>Often thought of as ceremonial and military implements first, perhaps the greatest contribution of elephants to human society has been as tremendous beasts of burden. Elephants are prized as a labour force even within today's lumber camps for their ability to haul the dense teak lumber found only within Burma and Thailand. The power of a fully grown elephant is matched only by its grace, as the animals can navigate difficult and uneven terrain impassable by carts and tractors while carrying loads far in excess of other animals. The use of elephants in logging also has had unintentional benefits to the health of the forests, with their delicate command of the forest trails reducing the potential damage of logging trails.[NEWLINE][NEWLINE]
+			The jobs of Southeast Asia's working elephants have come under threat within the past few decades with the mechanization of logging. Elephants and their mahouts have been forced to look for work in tourism and religious festivals, jobs which elephants find terribly stressful. </Text>
 		</Row>
 		
 		<!-- Bastu -->
 		<!-- Created by pineppledan 25/11/2017, modified and incorporated by adan_eslavo 17/12/2017 -->
 		<Row Tag="TXT_KEY_BUILDING_SWEDEN_BASTU">
-			<Text>Sauna</Text>
+			<Text>Bastu</Text>
```

Comment:
So we change it to Bastu? Fox said that Sauna is also good and easier for English speaker to understand.

Reply:
I guess we should stick to Sauna